inspire_json_merger/pre_filters.py

```diff
@@ -201,3 +205,16 @@ def clean_root_for_acquisition_source(root, head, update):
 
 filter_documents_same_source = partial(keep_only_update_source_in_field, 'documents')
 filter_figures_same_source = partial(keep_only_update_source_in_field, 'figures')
+
+
+def update_material(root, head, update):
+    if "erratum" in get_value(update, "dois.material"):
+        return
```

Comment:
```suggestion
        return root, head, update
```
probably (so I guess this is not tested :wink:)